t/Mock/MapIt.pm

```diff
@@ -115,6 +115,11 @@ sub dispatch_request {
             $self->output({2326 => {parent_area => undef, id => 2326, name => "Cheltenham Borough Council", type => "DIS"}});
         } elsif ($areas eq 'UTA') {
             $self->output({2650 => {parent_area => undef, id => 2650, name => "Aberdeen Council", type => "UTA"}});
+        } elsif ($areas eq 'LBO') {
+            $self->output({
+                2482 => {parent_area => undef, id => 2482, name => "Bromley Borough Council", type => "LBO"},
+                2483 => {parent_area => undef, id => 2483, name => "Hounslow Borough Council", type => "LBO"},
```

Comment:
Just to note another PR adds this but with slightly different councils, for any future merge/tidying.